src/networkLayer/client.js

```diff
@@ -0,0 +1,206 @@
+/* @flow */
+import Relay from 'react-relay/classic';
+import RequestObject from './requestObject';
+
+export type ClientNetworkLayerOpts = {
```

Comment:
A bunch of this stuff shouldn't be in relay-helpers - it's very Mixcloud specific